specs/latest/2.0/index.html

```diff
@@ -3159,6 +3159,15 @@ <h2 class="no-toc">Table of contents</h2>
         conversion to <em>internalformat</em>, but new components cannot be added.
     </p>
 
+    <h4>getFramebufferAttachmentParameter with FRAMEBUFFER_ATTACHMENT_OBJECT_NAME</h4>
+
+    <p>
+        If <code>getFramebufferAttachmentParameter</code> is called with <em>pname</em>
+        <code>FRAMEBUFFER_ATTACHMENT_OBJECT_TYPE</code> and <em>attachment</em> has no
```

Comment:
Did you mean to refer to FRAMEBUFFER_ATTACHMENT_OBJECT_NAME here, rather than FRAMEBUFFER_ATTACHMENT_OBJECT_TYPE?